config/flags_payments.go

```diff
@@ -139,6 +139,13 @@ var (
 		Value:  metadata.Testnet3Definition.Testnet3HermesURL,
 		Hidden: true,
 	}
+	// FlagPaymentsDuringSessionDebug sets if we're in debug more for the payments done in a VPN session.
+	FlagPaymentsDuringSessionDebug = cli.BoolFlag{
+		Name:   "payments.during-session-debug",
+		Usage:  "Set debug mode for payments made during a session, it will bypass any price validation and allow absurd prices during sessions",
```

Comment:
may users try to abuse it?

Reply:
Does it make a difference if it's us editing the code or them? Also not much you can abuse in mainnet really, you need real money to make money. So not sure how you would abuse this.